### PR TITLE
What_is_Encapsulation_in_cpp(A).md

### LINKED ISSUE #7567: What_is_Encapsulation_in_cpp(A).md
### Description

Ideation behind the concept of encapsulation in Object-Oriented Programming language

### Domain

C/CPP

### Type of Contribution

Audio

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by miraj0507**
<hr>

## Description 📜

What is Encapsulation in Object-Oriented Programming Languages?
Detailed discussion about the use of encapsulation to hide data.
Fixes #7567

<hr>

## Type of change 📝

- [x] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)

<hr>

## Domain of Contribution 📊

- [x] C/#CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Suvraneel** [comment]
@gunjan-g @ChetasShree

**Comment by Suvraneel** [comment]
@miraj0507  Kindly follow proper PR template directives from next time... You had to remove that hashtag from domain `C/CPP`... Consequently, this wasn't labelled as CPP... And it didn't show up in the mentors/sups feed

**Review by vasu-1 — CHANGES_REQUESTED** [review]
- Do some markdown Formatting !
- Add your md file link in the respective folder's `readme.md`


Audio is LGTM @miraj0507

**Comment by miraj0507** [comment]
> * Do some markdown Formatting !
> * Add your md file link in the respective folder's `readme.md`
> 
> Audio is LGTM @miraj0507

I have successfully made the changes

File: C_CPP/OOPs in C++/Encapsulation/What_is_Encapsulation_in_cpp(A).md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# WHAT IS ENCAPSULATION IN OBJECT ORIENTED PROGRAMMING LANGUAGES? 
+WHY THERE WAS A NEED FOR ENCAPSULATION?<br />
```
Comment (gunjan-g) [inline C_CPP/OOPs in C++/Encapsulation/What_is_Encapsulation_in_cpp(A).md:2]:
```suggestion
WHY THERE WAS A NEED FOR ENCAPSULATION?<br /> <br/>
```

File: C_CPP/OOPs in C++/Encapsulation/What_is_Encapsulation_in_cpp(A).md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+1)The concepts of hiding data with the help of encapsulation are going to be discussed in the audio.<br />
+2)How is it used as a mechanism to restrict certain action of classes outsite the scope are discussed in the audio.<br />
```
Comment (gunjan-g) [inline C_CPP/OOPs in C++/Encapsulation/What_is_Encapsulation_in_cpp(A).md:4]:
```suggestion
2)How is it used as a mechanism to restrict certain action of classes outsite the scope are discussed in the audio.<br /> <br/>
```

File: C_CPP/OOPs in C++/Encapsulation/What_is_Encapsulation_in_cpp(A).md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+[Getting started with oops? Understand Encapsulation.](https://drive.google.com/file/d/1WmftmYcTooSUnkA6lU3iOG3HfsXiVm0n/view?usp=sharing)
```
Comment (gunjan-g) [inline C_CPP/OOPs in C++/Encapsulation/What_is_Encapsulation_in_cpp(A).md:5]:
```suggestion
[Getting started with OOPS? Understand Encapsulation.](https://drive.google.com/file/d/1WmftmYcTooSUnkA6lU3iOG3HfsXiVm0n/view?usp=sharing)
```

**Review by gunjan-g — CHANGES_REQUESTED** [review]
@miraj0507 Please do the required changes

**Review by gunjan-g — APPROVED** [review]
@Suvraneel @Saviour1001 LGTM 🚀

**Pull request merged**